lib/services/rally/rally_release_resource.rb

```diff
@@ -1,3 +1,4 @@
+require 'active_support/core_ext'
```

Comment:
That dependency was put there because `1.day` was blowing up when used in isolation/testing. So, most likely, no. But it did cause me problems.